deploy/webhook/member-operator-webhook.yaml

```diff
@@ -95,6 +95,38 @@ objects:
     namespaceSelector:
       matchLabels:
         toolchain.dev.openshift.com/provider: codeready-toolchain
+- apiVersion: admissionregistration.k8s.io/v1
+  kind: ValidatingWebhookConfiguration
+  metadata:
+    name: member-operator-validating-webhook
+    labels:
+      app: member-operator-webhook
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  webhooks:
+    - name: users.rolebindings.webhook.sandbox
```

Comment:
```suggestion
    - name: users.rolebindings.validating.webhook
```

Reply:
@xcoulon I would keep sandbox specific name here. Also `users.rolebindings.webhook.sandbox` is consistent with `users.pods.webhook.sandbox` name we use for our user pod webhook.